src/MessageHandler/ActivityPub/UpdateActorHandler.php

```diff
@@ -22,8 +21,7 @@ class UpdateActorHandler extends MbinMessageHandler
 {
     public function __construct(
         private readonly EntityManagerInterface $entityManager,
-        private readonly KernelInterface $kernel,
```

Comment:
Ow I get:

```
{"message":"Warning: Undefined property: App\\MessageHandler\\ActivityPub\\UpdateActorHandler::$kernel","context":{"exception":{"class":"ErrorException","message":"Warning: Undefined property: App\\MessageHandler\\ActivityPub\\UpdateActorHandler::$kernel","code":0,"file":"/var/www/kbin.melroy.org/html/src/MessageHandler/ActivityPub/UpdateActorHandler.php:31"}},"level":400,"level_name":"ERROR","channel":"php","datetime":"2024-12-31T15:07:42.971401+01:00","extra":{}}
{"message":"Error thrown while handling message App\\Message\\ActivityPub\\UpdateActorMessage. Sending for retry #1 using 300000 ms delay. Error: \"App\\MessageHandler\\MbinMessageHandler::__construct(): Argument #2 ($kernel) must be of type Symfony\\Component\\HttpKernel\\KernelInterface, null given, called in /var/www/kbin.melroy.org/html/src/MessageHandler/ActivityPub/UpdateActorHandler.php on line 31\"","context":{"class":"App\\Message\\ActivityPub\\UpdateActorMessage","message_id":null,"retryCount":1,"delay":300000,"error":"App\\MessageHandler\\MbinMessageHandler::__construct(): Argument #2 ($kernel) must be of type Symfony\\Component\\HttpKernel\\KernelInterface, null given, called in /var/www/kbin.melroy.org/html/src/MessageHandler/ActivityPub/UpdateActorHandler.php on line 31","exception":{"class":"TypeError","message":"App\\MessageHandler\\MbinMessageHandler::__construct(): Argument #2 ($kernel) must be of type Symfony\\Component\\HttpKernel\\KernelInterface, null given, called in /var/www/kbin.melroy.org/html/src/MessageHandler/ActivityPub/UpdateActorHandler.php on line 31","code":0,"file":"/var/www/kbin.melroy.org/html/src/MessageHandler/MbinMessageHandler.php:14"}},"level":300,"level_name":"WARNING","channel":"messenger","datetime":"2024-12-31T15:07:42.971997+01:00","extra":{}}
{"message":"Warning: Undefined property: App\\MessageHandler\\ActivityPub\\UpdateActorHandler::$kernel","context":{"exception":{"class":"ErrorException","message":"Warning: Undefined property: App\\MessageHandler\\ActivityPub\\UpdateActorHandler::$kernel","code":0,"file":"/var/www/kbin.melroy.org/html/src/MessageHandler/ActivityPub/UpdateActorHandler.php:31"}},"level":400,"level_name":"ERROR","channel":"php","datetime":"2024-12-31T15:07:44.561503+01:00","extra":{}}
{"message":"Warning: Undefined property: App\\MessageHandler\\ActivityPub\\UpdateActorHandler::$kernel","context":{"exception":{"class":"ErrorException","message":"Warning: Undefined property: App\\MessageHandler\\ActivityPub\\UpdateActorHandler::$kernel","code":0,"file":"/var/www/kbin.melroy.org/html/src/MessageHandler/ActivityPub/UpdateActorHandler.php:31"}},"level":400,"level_name":"ERROR","channel":"php","datetime":"2024-12-31T15:07:42.971401+01:00","extra":{}}
{"message":"Error thrown while handling message App\\Message\\ActivityPub\\UpdateActorMessage. Sending for retry #1 using 300000 ms delay. Error: \"App\\MessageHandler\\MbinMessageHandler::__construct(): Argument #2 ($kernel) must be of type Symfony\\Component\\HttpKernel\\KernelInterface, null given, called in /var/www/kbin.melroy.org/html/src/MessageHandler/ActivityPub/UpdateActorHandler.php on line 31\"","context":{"class":"App\\Message\\ActivityPub\\UpdateActorMessage","message_id":null,"retryCount":1,"delay":300000,"error":"App\\MessageHandler\\MbinMessageHandler::__construct(): Argument #2 ($kernel) must be of type Symfony\\Component\\HttpKernel\\KernelInterface, null given, called in /var/www/kbin.melroy.org/html/src/MessageHandler/ActivityPub/UpdateActorHandler.php on line 31","exception":{"class":"TypeError","message":"App\\MessageHandler\\MbinMessageHandler::__construct(): Argument #2 ($kernel) must be of type Symfony\\Component\\HttpKernel\\KernelInterface, null given, called in /var/www/kbin.melroy.org/html/src/MessageHandler/ActivityPub/UpdateActorHandler.php on line 31","code":0,"file":"/var/www/kbin.melroy.org/html/src/MessageHandler/MbinMessageHandler.php:14"}},"level":300,"level_name":"WARNING","channel":"messenger","datetime":"2024-12-31T15:07:42.971997+01:00","extra":{}}
{"message":"Warning: Undefined property: App\\MessageHandler\\ActivityPub\\UpdateActorHandler::$kernel","context":{"exception":{"class":"ErrorException","message":"Warning: Undefined property: App\\MessageHandler\\ActivityPub\\UpdateActorHandler::$kernel","code":0,"file":"/var/www/kbin.melroy.org/html/src/MessageHandler/ActivityPub/UpdateActorHandler.php:31"}},"level":400,"level_name":"ERROR","channel":"php","datetime":"2024-12-31T15:07:44.561503+01:00","extra":{}}
```

So maybe it used after all.. At least not in the code, but it might be just a requirement.

Reply:
It was explained here by BentiGorlich: https://github.com/MbinOrg/mbin/pull/1300#issuecomment-2562302435

Reply:
@BentiGorlich I would expect my integration test fails due to removing `KernelInterface $kernel` in files like UpdateActorMessage and UpdateActorHandler.. But the integration test it **not** failing.